addons/dark-www/addon.json

```diff
@@ -2333,6 +2355,44 @@
         "darkForumCode": true
       }
     },
+    {
+      "name": "Scratch 2.0 (2013)",
```

Comment:
Could we rename this to just `Scratch 2.0`?
These colors are used to this day on some pages, and if someone wants a 2.0 preset, this one is probably the one they want

Reply:
The other one could be kept as `Scratch 2.0 (2015)` or possibly it could be renamed to `Scratch 2.0 (2015-2018)`

Reply:
> These colors are used to this day on some pages, and if someone wants a 2.0 preset, this one is probably the one they want

Well, I'm not the one to judge this, but I assume someone looking for the "2.0 preset" will want the navbar in the front page to have the exact same color as the navbar on My Stuff?